### DISCUSSION TITLE
Replaying endings of some beatmaps.

### DISCUSSION THREAD

**Discussion by Stocktheprism1** [ppy/osu#24001] [Q&A]
When at the end of a beatmap, it will replay the ending of the beatmap, usually under 2 seconds. In my findings it overrides the score for the notes replayed, but adds to spinners replayed. If I remember correctly, this also occurs in mania.

https://github.com/ppy/osu/assets/95490395/78396bb3-8f5f-4348-810c-6f150a39b68a

Example: I keep my combo until the end, I replay the same notes, and if I miss those notes, it will count as misses. If I spin a spinner, leaving a 500 spin bonus, then replay the spinner, I start at 500 and I am able to increase the bonus. The replayed section **does** affect the results screen

**Beatmaps With Problems I could find/remember:**

https://osu.ppy.sh/beatmapsets/1629379#osu/3414776
https://osu.ppy.sh/beatmapsets/979887#osu/2114276

**Logs**

_
[database.log](https://github.com/ppy/osu/files/11835730/database.log)
[input.log](https://github.com/ppy/osu/files/11835731/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11835732/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11835734/network.log)
[performance.log](https://github.com/ppy/osu/files/11835735/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11835736/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11835737/updater.log)
_

**OS**

Windows 11 Home
Version: 22H2

**Lazer Version**

2023.621.0

**Comment by peppy** [comment]
I can't reproduce this, so it looks like it may be something to do with hardware / sound card drivers.

**Reply by Stocktheprism1** [reply]
If I find anything that helps Ill make a post, thank you for trying to help :)
